hw2/05-form.js

```diff
@@ -0,0 +1,32 @@
+const registrationForm = document.querySelector('.registration-form');
+
+const logForm = function captureAndLogRegistrationForm(event) {
+  const {
+    fullName, email, registrationStatus, coursesTaken, anythingElse,
+  } = event.target;
+  let coursesNamesTaken = [...coursesTaken];
```

Comment:
I really like how you handled the coursesTaken data here, I always forget about the spread operator!